pkg/controller/import-controller.go

```diff
@@ -192,7 +192,7 @@ func (r *ImportReconciler) shouldReconcilePVC(pvc *corev1.PersistentVolumeClaim,
 	if err != nil {
 		return false, err
 	}
-	return !isPVCComplete(pvc) &&
+	return (!isPVCComplete(pvc) || isPodRetainAfterCompletion(pvc)) &&
```

Comment:
I am not quite sure why this || isPodRetainAfterCompletion is here. Am I missing something?

Reply:
Otherwise, when we manually delete the pvc/dv, it will prevent Reconcile() to call reconcilePvc() for the importer pod deletion.

Reply:
But isn't the pod owned by the PVC, so if the PVC is deleted, k8s (not our controller) will delete the pod.

Reply:
That's not the behavior I've seen, but I'll verify it.

Reply:
Verified. Pod is owned by PVC, but when PVC is deleted, Pod is deleted by our controller and NOT by k8s.
If it smells like a bug, we should fix it anyway in another card.

Reply:
Was pod phase `Succeeded` in your test?  If not, I can understand k8s not terminating a running pod

Reply:
You can see that if you comment out the explicit importer pod deletion for the case pvc was deleted (pvc.DeletionTimestamp != nil), the pod won't be deleted.

Reply:
Right, fixing it to `|| (isPodRetainAfterCompletion(pvc) && pvc.DeletionTimestamp != nil)`, which is better I guess?